ci-operator/step-registry/hypershift/mce/ibmz/create/hcp/hypershift-mce-ibmz-create-hcp-commands.sh

```diff
@@ -102,8 +102,10 @@ data:
 EOF
 
 # Creating AgentServiceConfig
+CLUSTER_VERSION=$(oc get clusterversion -o jsonpath={..desired.version} | cut -d '.' -f 1,2)
+OS_IMAGES=$(jq --arg CLUSTER_VERSION "${CLUSTER_VERSION}" '[.[] | select(.openshift_version == $CLUSTER_VERSION)]' "${SHARED_DIR}/default_os_images.json")
```

Comment:
What version do you select when you find multiple occurrences? (for instance 4.16 with multiple OS arch)

Reply:
Each version will only have one image.
https://github.com/openshift/assisted-service/blob/master/data/default_os_images.json